src/sentry/integrations/models/data_forwarding_config.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import DefaultFieldsModel, FlexibleForeignKey, region_silo_model
+from sentry.db.models.fields.jsonfield import JSONField
+
+
+@region_silo_model
+class DataForwardingConfig(DefaultFieldsModel):
```

Comment:
I probably set you up for this (mb) but I think the name data forwarding config should be something else, only because

```python
config = DataForwardingConfig.objects.get(...)
property = config.config.get(...) # this is kinda confusing
```

Maybe `DataForwardingIntegration` or `DataForwarder`?

Or you can change the actual field from `config` to `spec` or `setting`, just so they don't conflict.